service/src/main/java/com/epam/rft/atsy/service/impl/ApplicationsServiceImpl.java

```diff
@@ -67,7 +70,16 @@
 
     List<ApplicationEntity> applicationEntities = pageResult.getContent();
 
-    return converterService.convert(applicationEntities, ApplicationDTO.class);
+    List<CandidateApplicationDTO>
+        candidateApplicationDTOs =
+        converterService.convert(applicationEntities, CandidateApplicationDTO.class);
+    Long total = pageResult.getTotalElements();
+
+    PagingResponse<CandidateApplicationDTO>
+        response =
+        new PagingResponse<>(total, candidateApplicationDTOs);
```

Comment:
Local variable "response" maybe not important, you can return with new PaginResponse  object.